addons/project_timesheet_holidays/tests/test_timesheet_holidays.py

```diff
@@ -238,8 +238,8 @@ def test_delete_timesheet_after_new_holiday_covers_whole_timeoff(self):
             'name': 'Test Time off please',
             'employee_id': self.empl_employee.id,
             'holiday_status_id': self.hr_leave_type_with_ts.id,
-            'date_from': leave_start_datetime,
-            'date_to': leave_end_datetime,
+            'request_date_from': leave_start_datetime,
```

Comment:
It would be nice to have a test showing reproducing the issue and showing your fix solves the issue. 🙂

Reply:
`test_delete_timesheet_after_new_holiday_covers_whole_timeoff` is added for this purpose in this PR, so I believe the issue is tested. Thanks!